src/stream_processor/parser/sql.y

```diff
@@ -276,7 +275,19 @@ select: SELECT keys FROM source window where groupby limit ';'
               {
                 flb_sp_cmd_window(cmd, FLB_SP_WINDOW_HOPPING, $3, $4, $7, $8);
               }
-      condition: comparison
+      condition: condition OR condition1
+                 {
+                   $$ = flb_sp_cmd_operation(cmd, $1, $3, FLB_EXP_OR);
+                 }
+                 |
+                 condition1
+      condition1: condition1 AND condition2
+                  {
+                    $$ = flb_sp_cmd_operation(cmd, $1, $3, FLB_EXP_AND);
+                  }
+                  |
+                  condition2
+      condition2: comparison
```

Comment:
let's keep the name `condition` here, and rename `condition` and `condition1` as suggested.